waspc/src/Wasp/Generator/WriteFileDrafts.hs

```diff
@@ -31,6 +31,10 @@ import Wasp.Util (Checksum)
 -- It is smart when writing, so it doesn't write file drafts that are already written on the disk from before.
 synchronizeFileDraftsWithDisk :: Path' Abs (Dir ProjectRootDir) -> [FileDraft] -> IO ()
 synchronizeFileDraftsWithDisk dstDir fileDrafts = do
+
+  -- If the destination file paths are not unique, throw error
```

Comment:
For comments, we typically end them with a period. Also, you may be able to remove this one at the call site even and rely on the one above the function.